contracts/src/core/IAlignedLayerServiceManager.sol

```diff
@@ -62,4 +65,12 @@ interface IAlignedLayerServiceManager {
     function balanceOf(address account) external view returns (uint256);
 
     function setAggregator(address _aggregator) external;
+
+    function isVerifierBlacklisted(
+        uint256 verifierIdx
+    ) external view returns (bool);
+    function blacklistVerifier(uint256 verifierIdx) external;
+    function whitelistVerifier(uint256 verifierIdx) external;
+    function setVerifiersBlacklist(uint256 bitmap) external;
+    function getBlacklistedVerifiers() external view returns (uint256);
```

Comment:
Redundant function since the variable is already public

Reply:
True, fixed!